source/deployment/admin-roles.rst

```diff
@@ -59,15 +59,38 @@ The format of the mmctl command is:
 
 ``mmctl permissions role assign system_user_manager bob-smith sue-clark``
 
-``mmctl permissions role assign system_read_only_admin captain-marvel clark-kent``
+**To grant the Read Only Admin role to two users, Bob Smith and Sue Clark:**
+
+``mmctl permissions role assign system_read_only_admin bob-smith sue-clark``
+
+Editing Privileges of Admin Roles (Advanced)
+--------------------------------------------
+
+Each of the admin roles have defined, default privileges as outlined above. However both read and write access to areas of the System Console can be granted or removed from each role by System Admins using the mmctl tool. This can be done either locally or remotely.
```

Comment:
Does this mean that although there are default settings, those defaults can be modified? For example, can the user manager role be given more access over and above the default access? Can the System Manager have some access removed? If so, is there a table indicating which defaults have been modified?

Reply:
Yes, that is what that means. No, there is not currently a way to see what has been changed from defaults,  but the UI (coming in a future release) will eventually reflect that. We are also planning to add a "reset to defaults" switch.

Reply:
Awesome, thank you!